frontend/src/metabase-lib/lib/queries/structured/Join.js

```diff
@@ -192,6 +203,17 @@ export default class Join extends MBQLObjectClause {
     return this.setAlias(alias);
   }
 
+  getConditions() {
+    if (!this.condition) {
+      return [];
+    }
+    if (this.isSingleConditionJoin()) {
+      return [this.condition];
+    }
+    const [_operator, ...conditions] = this.condition;
```

Comment:
you can do `const [, ...conditions] = this.condition;` and that'll avoid the need for disabling eslint's no-unused-vars

Reply:
Thanks for this tip.
I didn't know this is possible and was avoiding spread operator on at least a few occasions because of linting.

Reply:
Thank you, Dalton! Didn't know it's possible, it's so handy

Reply:
yep, it is pretty flexible. you can do multiple commas in a row, too.

```
const foo = [1, 2, 3, 4];
const [one, , , four] = foo;
```